src/amalgam/gen/wasmsimd.c

```diff
@@ -27266,6 +27266,56 @@ void xnn_f32_vrndz_ukernel__wasmsimd_u8(
   }
 }
 
+void xnn_f32_vrsqrt_ukernel__wasmsimd_rsqrt_u8(
+    size_t batch,
+    const float* input,
+    float* output,
+    const union xnn_f32_rsqrt_params params[restrict XNN_MIN_ELEMENTS(1)]) XNN_OOB_READS
+{
+  assert(batch != 0);
+  assert(batch % sizeof(float) == 0);
+  assert(input != NULL);
+  assert(output != NULL);
+
+  const v128_t vone = wasm_f32x4_const_splat(1.0f);
+
+  for (; batch >= 8 * sizeof(float); batch -= 8 * sizeof(float)) {
+    const v128_t vx0123 = wasm_v128_load(input);
+    const v128_t vx4567 = wasm_v128_load(input + 4);
+    input += 8;
+
+    const v128_t vt0123 = wasm_f32x4_sqrt(vx0123);
+    const v128_t vt4567 = wasm_f32x4_sqrt(vx4567);
+    const v128_t vy0123 = wasm_f32x4_div(vone, vt0123);
```

Comment:
some cpus dont support div, such as arm.  Is there an alternative way to get reciprocal?

Reply:
Not really, since WASM SIMD does not support `rcp` or `rsqrt` approximations, since they would not be identical across all platforms.

We could compute an `rsqrt` approximation ourselves, e.g. a polynomial, but I doubt that would be faster than whatever WASM would to under the hood here.